scripts/setup-adapters.sh

```diff
@@ -61,7 +61,14 @@ cd "${DEPENDENCY_DIR}" || exit
 # aws-sdk-cpp missing dependencies
 
 if [[ "$OSTYPE" == "linux-gnu"* ]]; then
-   yum -y install libxml2-devel libgsasl-devel libuuid-devel
+   # /etc/os-release is a standard way to query various distribution
+   # information and is available everywhere
+   LINUX_DISTRIBUTION=$(cat /etc/os-release | awk -F= '/^ID=/ {print $2}')
```

Comment:
I haven't done bash in a while, but given the layout of this file, wouldn't something like this be the way to parse it?

> LINUX_DISTRIBUTION=$(. /etc/os-release && echo ${ID})

Reply:
I prefer this suggestion.

Reply:
@pedroerp Indeed, your suggestion seems to be better. I'll fix that.